pkg/api/types.go

```diff
@@ -12,6 +14,24 @@ const (
 	KubernetesNamespace = "io.k8s."
 )
 
+const (
+	// PullAlways means that we always attempt to pull the latest image.
+	PullAlways PullPolicy = "always"
+
+	// PullNever means that we never pull an image, but only use a local image.
+	PullNever PullPolicy = "never"
+
+	// PullIfNotPresent means that we pull if the image isn't present on disk.
+	PullIfNotPresent PullPolicy = "if-not-present"
+
+	// DefaultBuilderPullPolicy specifies the default pull policy to use
+	DefaultBuilderPullPolicy = PullIfNotPresent
+
+	// DefaultPreviousImagePullPolicy specifies policy for pulling the previously
+	// build Docker image when doing incremental build
+	DefaultPreviousImagePullPolicy = PullAlways
```

Comment:
@soltysh so if you

1) come up with a use-case where setting this is actually useful
2) and give me reasonable option name

I can expose it to users :-)